django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -157,4 +157,25 @@
     window.showAddAnotherPopup = showRelatedObjectPopup;
     window.dismissAddAnotherPopup = dismissAddRelatedObjectPopup;
 
-})();
+    $(document).ready(function() {
```

Comment:
Not sure if we prefer one or the other but `$(function(){})` is equivalent to `$(document).ready(function(){})`.